### ISSUE TITLE
can't launch chrome

### ISSUE THREAD

**Issue by salomegeo** [willscott/grunt-jasmine-chromeapp#2]
I'm running this on linux. The error is "Starting Chrome...>> Timed out" and it never exits.

I have chrome installed, it's where it's expected to be at /usr/bin/google-chrome.

**Comment by willscott** [comment]
platform?

can you try running it yourself?
https://github.com/willscott/grunt-jasmine-chromeapp/blob/master/lib/launch-chrome.js#L30

what happens when you run via `grunt jasmine_chromeapp --debug --verbose`

I'm not sure what else I can do to debug.

**Comment by trevj** [comment]
I'm trying to install this in uproxy-networking and this is the output from `grunt jasmine_chromeapp:base --debug --verbose`:

```
Running "jasmine_chromeapp:base" (jasmine_chromeapp) task
[D] Task source: /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/tasks/jasmine-chromeapp.js
Verifying property jasmine_chromeapp.base exists in config...OK
Files: build/integration/socks-echo/integration.js, build/integration/socks-echo/integration.js.map, build/integration/socks-echo/integration.json, build/integration/socks-echo/integration.ts, build/integration/tcp/integration.js, build/integration/tcp/integration.js.map, build/integration/tcp/integration.json, build/integration/tcp/integration.ts
Files: build/logging/logging.js
Files: build/handler/queue.js
Files: build/ipaddrjs/ipaddr.min.js
Files: build/rtc-to-net/rtc-to-net.js
Files: build/socks-common/socks-headers.js
Files: build/socks-to-rtc/socks-to-rtc.js
Files: build/webrtc/datachannel.js, build/webrtc/peerconnection.js
Files: build/tcp/tcp.js
Files: build/freedom/freedom-for-chrome.js
Files: build/arraybuffers/arraybuffers.js
Options: template="/usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/tasks/../tasks/jasmine-chromeapp", version="2.0.0", outfile=".build", paths=["build/freedom/freedom-for-chrome.js","build/arraybuffers/arraybuffers.js"], binary=undefined, keepRunner, port=9989, timeout=30000
[D] {"template":"/usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/tasks/../tasks/jasmine-chromeapp","version":"2.0.0","outfile":".build","paths":["build/freedom/freedom-for-chrome.js","build/arraybuffers/arraybuffers.js"],"keepRunner":true,"port":9989,"timeout":30000}
Building...Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/tasks/jasmine-chromeapp/background.js...OK
Writing .build/background.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/tasks/jasmine-chromeapp/main.html...OK
Writing .build/main.html...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/tasks/jasmine-chromeapp/manifest.json...OK
Writing .build/manifest.json...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/tasks/jasmine-chromeapp/relay.js...OK
Writing .build/relay.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/__init__.py...OK
Writing .build/jasmine-core//__init__.py...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/boot/boot.js...OK
Writing .build/jasmine-core/boot/boot.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/boot.js...OK
Writing .build/jasmine-core//boot.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/core.py...OK
Writing .build/jasmine-core//core.py...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/example/spec/PlayerSpec.js...OK
Writing .build/jasmine-core/example/spec/PlayerSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/example/spec/SpecHelper.js...OK
Writing .build/jasmine-core/example/spec/SpecHelper.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/example/src/Player.js...OK
Writing .build/jasmine-core/example/src/Player.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/example/src/Song.js...OK
Writing .build/jasmine-core/example/src/Song.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/jasmine-html.js...OK
Writing .build/jasmine-core//jasmine-html.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/jasmine.css...OK
Writing .build/jasmine-core//jasmine.css...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/jasmine.js...OK
Writing .build/jasmine-core//jasmine.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/json2.js...OK
Writing .build/jasmine-core//json2.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/console/ConsoleReporterSpec.js...OK
Writing .build/jasmine-core/spec/console/ConsoleReporterSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/AnySpec.js...OK
Writing .build/jasmine-core/spec/core/AnySpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/CallTrackerSpec.js...OK
Writing .build/jasmine-core/spec/core/CallTrackerSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/ClockSpec.js...OK
Writing .build/jasmine-core/spec/core/ClockSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/DelayedFunctionSchedulerSpec.js...OK
Writing .build/jasmine-core/spec/core/DelayedFunctionSchedulerSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/EnvSpec.js...OK
Writing .build/jasmine-core/spec/core/EnvSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/ExceptionFormatterSpec.js...OK
Writing .build/jasmine-core/spec/core/ExceptionFormatterSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/ExceptionsSpec.js...OK
Writing .build/jasmine-core/spec/core/ExceptionsSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/ExpectationResultSpec.js...OK
Writing .build/jasmine-core/spec/core/ExpectationResultSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/ExpectationSpec.js...OK
Writing .build/jasmine-core/spec/core/ExpectationSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/JsApiReporterSpec.js...OK
Writing .build/jasmine-core/spec/core/JsApiReporterSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/MockDateSpec.js...OK
Writing .build/jasmine-core/spec/core/MockDateSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/ObjectContainingSpec.js...OK
Writing .build/jasmine-core/spec/core/ObjectContainingSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/PrettyPrintSpec.js...OK
Writing .build/jasmine-core/spec/core/PrettyPrintSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/QueueRunnerSpec.js...OK
Writing .build/jasmine-core/spec/core/QueueRunnerSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/ReportDispatcherSpec.js...OK
Writing .build/jasmine-core/spec/core/ReportDispatcherSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/SpecSpec.js...OK
Writing .build/jasmine-core/spec/core/SpecSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/SpySpec.js...OK
Writing .build/jasmine-core/spec/core/SpySpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/SpyStrategySpec.js...OK
Writing .build/jasmine-core/spec/core/SpyStrategySpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/SuiteSpec.js...OK
Writing .build/jasmine-core/spec/core/SuiteSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/TimerSpec.js...OK
Writing .build/jasmine-core/spec/core/TimerSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/UtilSpec.js...OK
Writing .build/jasmine-core/spec/core/UtilSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/integration/CustomMatchersSpec.js...OK
Writing .build/jasmine-core/spec/core/integration/CustomMatchersSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/integration/EnvSpec.js...OK
Writing .build/jasmine-core/spec/core/integration/EnvSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/integration/SpecRunningSpec.js...OK
Writing .build/jasmine-core/spec/core/integration/SpecRunningSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/matchersUtilSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/matchersUtilSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toBeCloseToSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toBeCloseToSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toBeDefinedSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toBeDefinedSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toBeFalsySpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toBeFalsySpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toBeGreaterThanSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toBeGreaterThanSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toBeLessThanSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toBeLessThanSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toBeNaNSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toBeNaNSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toBeNullSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toBeNullSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toBeSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toBeSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toBeTruthySpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toBeTruthySpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toBeUndefinedSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toBeUndefinedSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toContainSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toContainSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toEqualSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toEqualSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toHaveBeenCalledSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toHaveBeenCalledSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toHaveBeenCalledWithSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toHaveBeenCalledWithSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toMatchSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toMatchSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toThrowErrorSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toThrowErrorSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/core/matchers/toThrowSpec.js...OK
Writing .build/jasmine-core/spec/core/matchers/toThrowSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/helpers/BrowserFlags.js...OK
Writing .build/jasmine-core/spec/helpers/BrowserFlags.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/helpers/defineJasmineUnderTest.js...OK
Writing .build/jasmine-core/spec/helpers/defineJasmineUnderTest.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/html/HtmlReporterSpec.js...OK
Writing .build/jasmine-core/spec/html/HtmlReporterSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/html/HtmlSpecFilterSpec.js...OK
Writing .build/jasmine-core/spec/html/HtmlSpecFilterSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/html/MatchersHtmlSpec.js...OK
Writing .build/jasmine-core/spec/html/MatchersHtmlSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/html/PrettyPrintHtmlSpec.js...OK
Writing .build/jasmine-core/spec/html/PrettyPrintHtmlSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/html/QueryStringSpec.js...OK
Writing .build/jasmine-core/spec/html/QueryStringSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/html/ResultsNodeSpec.js...OK
Writing .build/jasmine-core/spec/html/ResultsNodeSpec.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/javascripts/support/jasmine_selenium_runner.yml...OK
Writing .build/jasmine-core/spec/javascripts/support/jasmine_selenium_runner.yml...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/node_suite.js...OK
Writing .build/jasmine-core/spec/node_suite.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/performance/large_object_test.js...OK
Writing .build/jasmine-core/spec/performance/large_object_test.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/performance/performance_test.js...OK
Writing .build/jasmine-core/spec/performance/performance_test.js...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/support/jasmine-performance.yml...OK
Writing .build/jasmine-core/spec/support/jasmine-performance.yml...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/spec/support/jasmine.yml...OK
Writing .build/jasmine-core/spec/support/jasmine.yml...OK
Reading /usr/local/google/home/trevj/src/uproxy-networking/node_modules/grunt-jasmine-chromeapp/vendor/jasmine-core-2.0.0/version.rb...OK
Writing .build/jasmine-core//version.rb...OK
Reading build/integration/socks-echo/integration.js...OK
Writing .build/scripts/build/integration/socks-echo/integration.js...OK
Reading build/integration/socks-echo/integration.js.map...OK
Writing .build/scripts/build/integration/socks-echo/integration.js.map...OK
Reading build/integration/socks-echo/integration.json...OK
Writing .build/scripts/build/integration/socks-echo/integration.json...OK
Reading build/integration/socks-echo/integration.ts...OK
Writing .build/scripts/build/integration/socks-echo/integration.ts...OK
Reading build/integration/tcp/integration.js...OK
Writing .build/scripts/build/integration/tcp/integration.js...OK
Reading build/integration/tcp/integration.js.map...OK
Writing .build/scripts/build/integration/tcp/integration.js.map...OK
Reading build/integration/tcp/integration.json...OK
Writing .build/scripts/build/integration/tcp/integration.json...OK
Reading build/integration/tcp/integration.ts...OK
Writing .build/scripts/build/integration/tcp/integration.ts...OK
Reading build/logging/logging.js...OK
Writing .build/scripts/build/logging/logging.js...OK
Reading build/handler/queue.js...OK
Writing .build/scripts/build/handler/queue.js...OK
Reading build/ipaddrjs/ipaddr.min.js...OK
Writing .build/scripts/build/ipaddrjs/ipaddr.min.js...OK
Reading build/rtc-to-net/rtc-to-net.js...OK
Writing .build/scripts/build/rtc-to-net/rtc-to-net.js...OK
Reading build/socks-common/socks-headers.js...OK
Writing .build/scripts/build/socks-common/socks-headers.js...OK
Reading build/socks-to-rtc/socks-to-rtc.js...OK
Writing .build/scripts/build/socks-to-rtc/socks-to-rtc.js...OK
Reading build/webrtc/datachannel.js...OK
Writing .build/scripts/build/webrtc/datachannel.js...OK
Reading build/webrtc/peerconnection.js...OK
Writing .build/scripts/build/webrtc/peerconnection.js...OK
Reading build/tcp/tcp.js...OK
Writing .build/scripts/build/tcp/tcp.js...OK
Reading build/freedom/freedom-for-chrome.js...OK
Writing .build/scripts/build/freedom/freedom-for-chrome.js...OK
Reading build/arraybuffers/arraybuffers.js...OK
Writing .build/scripts/build/arraybuffers/arraybuffers.js...OK
Reading .build/main.html...OK
Writing .build/main.html...OK
Done.
Starting Reporter...Done.
Starting Chrome...>> Timed out
```

This is Ubuntu 14.04.1.

`/usr/local/google-chrome` exists.

**Comment by willscott** [comment]
This seems to have to do with the set of command line arguments passed to
chrome at:
https://github.com/willscott/grunt-jasmine-chromeapp/blob/master/tasks/jasmine-chromeapp.js#L111

See if removing the first 4 flags fixes this behavior, and if so whether
there's a single one that seems to prevent startup.

--Will

**Comment by trevj** [comment]
`--no-startup-window` seems to be the culprit.

**Comment by willscott** [comment]
Addressed in v1.4.0 https://github.com/willscott/grunt-jasmine-chromeapp/releases/tag/v1.4.0
